### PR TITLE
docs(community): adds links to videos from OpenDev conf 2018

### PR DESCRIPTION
**Pull request by dorbin**

### CODE REVIEW

File: community/presentations.md [hunk 1]
```diff
@@ -8,16 +8,22 @@ redirect_from: /docs/presentations
 
 Conference and Meetup Presentations on Spinnaker
 
+### <a href="http://2018.opendevconf.com/" target="_blank">OpenDev CI/CD</a>
+**Videos from OpenDev Conference, Vancouver, BC** \| May 22-23, 2018
+
+* [How Dev/Ops Evolves as the Cloud Does](https://www.youtube.com/watch?v=LjjfWBrvyps){:target="\_blank"}
```
Comment (demobox) [inline community/presentations.md]: I think this is "DevOps" (no `/`)? Also, curious about`target="_blank"`? The videos further down the page don't open in a new window, as far as I can make out?

/cc @skim1420

File: community/presentations.md [hunk 2]
```diff
@@ -8,16 +8,22 @@ redirect_from: /docs/presentations
...
+* [Open CD for Open Infrastructure Hybrid and Multi-Cloud Deployments with Spinnaker](https://www.youtube.com/watch?v=EC_zxrk2NQc){:target="\_blank"}
```
Comment (demobox) [inline community/presentations.md]: "...Infrastructure: Hybrid..." (add `:`)

**Review by demobox — CHANGES_REQUESTED** [review]
Thanks for adding these, Dave!

File: community/presentations.md (hunk 1 above)
Comment (skim1420) [reply community/presentations.md]: Yeah, omit the slash. slash

Comment (dorbin) [reply community/presentations.md]: @demobox , I've been adding the target="_blank", per #743 
It's not non-controversial ;), and I'm open to argument. 
Mean time, I'll add those on this page to be consistent.

**Pull request closed without merging**